napari/_qt/layer_controls/qt_surface_controls.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, layer):
         self.grid_layout.addWidget(self.contrastLimitsSlider, 1, 1)
         self.grid_layout.addWidget(QLabel('gamma:'), 2, 0)
         self.grid_layout.addWidget(self.gammaSlider, 2, 1)
-        self.grid_layout.addWidget(QLabel('colormap:'), 3, 0)
+        self.grid_layout.addWidget(self.colormapLabel, 3, 0)
```

Comment:
I believe this is the line causing issue, there is no colormapLabel in the object, you may want to revert it